framework/src/modules/interoperability/base_interoperability_store.ts

```diff
@@ -330,21 +350,18 @@ export abstract class BaseInteroperabilityStore {
 			return;
 		}
 
-		const interoperableModule = this._interoperableModules.get(ccm.moduleID);
-
+		const ccCommands = interoperableCCCommands.get(ccm.moduleID);
 		// When moduleID is not supported
```

Comment:
comment needs to be updated

Reply:
when ccCommands is undefined means that module entry doesn't exist, the next condition is to check ccCMDID